easybuild/easyblocks/generic/perlmodule.py

```diff
@@ -58,16 +58,17 @@ def install_perl_module(self):
         """Install procedure for Perl modules: using either Makefile.Pl or Build.PL."""
         # Perl modules have two possible installation procedures: using Makefile.PL and Build.PL
         # configure, build, test, install
+        unset_cmd = "unset PERL_MM_OPT PERL_MB_OPT"
```

Comment:
You'll probably need to make this `unset ... && ` to chain it with the Perl command? `preconfigopts` complicates this though...

Reply:
A better approach would be to use `unset_env_vars(['PERL_MB_OPT', 'PERL_MM_OPT'])`, where `unset_env_vars` can be imported from `easybuild.tools.environment`.

This will result in proper logging for example, and since the environment in which commands are run via `run_cmd` is inherited, that should work.

Reply:
Do include a comment to explain why these environment variables need to be unset.